moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/usePreviousData.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useMemo } from "react";
+
+/**
+ * Provide a constant stream of data from a useQuery hook to mask the loading state
+ * @param {Object} data - response from Apollo useQuery
+ * @returns {Object} - most current data from useQuery
+ */
+export const useCurrentData = (data) => {
+  const [previousData, setPreviousData] = useState(null);
+
+  const currentData = useMemo(() => {
+    if (data) {
+      setPreviousData(data);
+      return data;
+    } else {
+      return previousData;
+    }
+  }, [data, setPreviousData, previousData]);
+
+  return currentData;
+};
```

Comment:
this is working great!

one side effect of this is that the loading spinner no longer shows when sorting columns, searching, or advance searching. if you throttle your network down to 3G you can see this is probably gonna be a little laggy/jumpy in production.

one idea we discussed is showing the loading spinner elsewhere on the screen, so that we still give visual feedback when the table is updating. i will open a separate issue for that.